ytl-linux-cpu-governor/README.md

```diff
@@ -0,0 +1,10 @@
+# ytl-linux-cpu-governor
+
+This package depends on `cpufrequtils`, which sets CPU governor "performance" for
```

Comment:
which sets CPU governor "performance"
=>
**and** sets CPU governor "performance"